### PR TITLE
Fix shader format

### PR DESCRIPTION
**Pull request by JaimeBea**

### CODE REVIEW

File: Game/Shaders/phong_pbr_fragment.glsl [hunk 1]
```diff
@@ -39,127 +36,118 @@ struct PointLight
...
-{    
-	vec3 frag_n = normalize(frag_normal);
-	vec3 view_n = normalize(view_pos - frag_pos);
+void main() {    
+	vec3 fragN = normalize(fragNormal);
+	vec3 viewN = normalize(viewPos - fragPos);
 	
 	// diffuse 
-	vec3 diffuse_color = has_diffuse_map * pow(texture(diffuse_map, vec2(uv.x, 1 - uv.y)).rgb, vec3(2.2)) + (1 - has_diffuse_map) * diffuse_color;
+	vec3 diffuseColor = hasDiffuseMap * pow(texture(diffuseMap, vec2(uv.x, 1 - uv.y)).rgb, vec3(2.2)) + (1 - hasDiffuseMap) * diffuseColor;
```
Comment (Jordi-Gil) [inline Game/Shaders/phong_pbr_fragment.glsl:68]: Y-axis flipped twice? In the [texture loader](https://github.com/TBD-org/TBD-Engine/blob/4d9a7a9d7cfeedcf00533581817745465532f508/Project/Source/FileSystem/TextureImporter.cpp#L45) via devIL, the texture is flipped.

File: Game/Shaders/default_fragment.glsl [hunk 2]
```diff
@@ -1,12 +1,11 @@
...
 
 in vec2 uv0;
 
 out vec3 color;
 
-void main()
-{
-	color = texture(diffuse_texture, uv0).xyz;
+void main() {
+	color = texture(diffuseTexture, uv0).xyz;
```
Comment (Jordi-Gil) [inline Game/Shaders/default_fragment.glsl:10]: Change to .rgb for consistency. Is the same, but for colours is more verbose set .rgb.

**Review by Jordi-Gil — CHANGES_REQUESTED** [review]

File: Game/Shaders/phong_pbr_fragment.glsl (hunk 1 above)
Comment (JaimeBea) [reply Game/Shaders/phong_pbr_fragment.glsl:68]: This is because we have the textures flipped. This is a separate issue. I will add it to issues.

Comment (Jordi-Gil) [reply Game/Shaders/phong_pbr_fragment.glsl:68]: Perfect.

**Review by Jordi-Gil — APPROVED** [review]

**Pull request closed without merging**